pkg/api/rpms.go

```diff
@@ -51,6 +51,13 @@ type SnapshotSearchRpmRequest struct {
 	Limit  *int     `json:"limit,omitempty"` // Maximum number of records to return for the search
 }
 
+type DetectRpmsRequest struct {
+	URLs     []string `json:"urls,omitempty"`  // URLs of repositories to search
+	UUIDs    []string `json:"uuids,omitempty"` // List of RepositoryConfig UUIDs to search
```

Comment:
```suggestion
	UUIDs    []string `json:"uuids,omitempty"` // List of repository UUIDs to search
```
we should say "repository" here since "repository config" is not a user-facing concept